renpy/savelocation.py

```diff
@@ -287,13 +287,9 @@ def rename(self, old, new):
             if not os.path.exists(old):
                 return
 
-            os.rename(old, old + ".tmp")
-            old = old + ".tmp"
-
-            if os.path.exists(new):
-                os.unlink(new)
-
-            os.rename(old, new)
+            old_tmp = old + tmp
```

Comment:
You probably mean `".tmp"` instead of `tmp`

Reply:
It is the same as below with persistent